core/c/libc/complex.odin

```diff
@@ -3,7 +3,11 @@ package libc
 // 7.3 Complex arithmetic
 
 when ODIN_OS == .Windows {
-	foreign import libc "system:libucrt.lib"
+	when ODIN_CRT_MD {
```

Comment:
Why is it called `ODIN_CRT_MD`?

Reply:
I'm open for a better name. The current name is just the `ODIN` prefix followed by the subject `CRT` and `MD` for the C++ equivalent `/MD` flag.

Reply:
How about `ODIN_DYNAMIC_CRT` which is what `/MD` does as opposed to `/MT` that statically link CRT?

Reply:
`DYNAMIC_CRT` is certainly better than `CRT_MD`.